core/market/src/db/model/agreement.rs

```diff
@@ -14,42 +8,45 @@ use ya_client::model::market::agreement::{
 use ya_client::model::market::demand::Demand as ClientDemand;
 use ya_client::model::market::offer::Offer as ClientOffer;
 use ya_client::model::{ErrorMessage, NodeId};
+use ya_diesel_utils::DatabaseTextField;
 
 use crate::db::model::{OwnerType, Proposal, ProposalId, SubscriptionId};
 use crate::db::schema::market_agreement;
 
 pub type AgreementId = ProposalId;
+pub type AppSessionId = Option<String>;
```

Comment:
as this is user input, we need to check if this is at most 100 chars. Is there a unit test for such a case?

Reply:
I didn't know, we have such a limit. Is it documented?

Reply:
you have implicitly set this limit via `up.sql` ;)